frontend/src/v5/ui/components/shared/sso/ssoLinkingResponseHandler/ssoLinkingResponseHandler.component.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { DialogsActionsDispatchers } from '@/v5/services/actionsDispatchers';
+import { CurrentUserHooksSelectors } from '@/v5/services/selectorsHooks';
+import { EditProfileModal } from '@components/shared/userMenu/editProfileModal/editProfileModal.component';
+import { useEffect } from 'react';
+import { useSSO } from '../useSSO';
+
+export const SSOResponseHandler = () => {
+	const { username } = CurrentUserHooksSelectors.selectCurrentUser();
+	const { linkPost, unlinkPost } = useSSO();
+
+	useEffect(() => {
+		const isSSOLinkAction = linkPost || unlinkPost;
+		if (!username || !isSSOLinkAction) return;
+		DialogsActionsDispatchers.open(EditProfileModal, { initialTab: 'authentication' });
```

Comment:
What does this component do?

Reply:
shows the edit profile modal in the authentication tab if there was a link/unlink post action